### PR TITLE
Convert markdown files to Dactyl style markdown

### PR DESCRIPTION
**Pull request by pgarg-ripple**

### CODE REVIEW

File: docs/markdown/connecting-to-testnet.md [hunk 1]
```diff
@@ -8,20 +8,23 @@ This tutorial describes how to:
...
 
-NOTE: Even though Interledger supports many different underlying asset types, you do not need to use real 
+**Note:** Even though Interledger supports many different underlying asset types, you do not need to use real 
 cryptocurrency for this tutorial. Since you will be sending value through an ILP router on the testnet, 
 funds will be sourced from the [XRP testnet faucet](https://xrpl.org/xrp-test-net-faucet.html) and 
 will have no real-world value.
 
+
 ## Configuring moneyd
 NOTE: Before configuring `moneyd`, make sure to delete the `.moneyd.test.json` config file, located in the home folder
```
Comment (ryangyoung) [inline docs/markdown/connecting-to-testnet.md]: New line here, too, please. Also, to be consistent the note should match the styling in line 22, i.e.:

> **Note:** Before configuring `moneyd`, make sure to delete the `.moneyd.test.json` config file, located in the home folder of your computer, if the file already exists. 

Also, another line break between 29 and 30.

Comment (pgarg-ripple) [reply docs/markdown/connecting-to-testnet.md]: https://github.com/interledger/interledger.github.io/pull/115/files#diff-95b449acc2e692458b91372a5b6e33e7R32

File: docs/markdown/getting-started.md [hunk 2]
```diff
@@ -3,18 +3,19 @@
...
 
 ## Before you begin
 Install a stable version of [Node.js](https://nodejs.org/en/) (10.16.0 LTS is recommended)
 
-NOTE: For this tutorial you *do not* need to use any cryptocurrency. Since you will be running ILP 
+**Note:** For this tutorial you *do not* need to use any cryptocurrency. Since you will be running ILP 
 on a local test network, settlement (moving real money) does not take place. 
 
+
 ## Installing moneyd
```
[the remark this answers is not in the document]
Comment (pgarg-ripple) [reply docs/markdown/getting-started.md]: Done.

File: docs/markdown/getting-started.md [hunk 3]
```diff
@@ -3,18 +3,19 @@
...
 To install `moneyd`, open a terminal and run the following command:
 ```shell
```
[the remark this answers is not in the document]
Comment (pgarg-ripple) [reply docs/markdown/getting-started.md]: https://github.com/interledger/interledger.github.io/pull/115/files#diff-95b449acc2e692458b91372a5b6e33e7R38

**Pull request merged**